README.md

```diff
@@ -1,18 +1,26 @@
 # nebula-spark-utils
 
-The nebula-spark-utils repository holds all the Apache Spark&trade; related libraries and tools for Nebula Graph v2.x.
+> **说明**:
 
-There are the version correspondence between Nebula Spark utils and Nebula:
+nebula-spark-utils 仓库包含Nebula的三个产品：Nebula Exchange、Nebula Spark Connector 和 Nebula Algorithm。
 
-| Nebula Exchange Version | Nebula Spark Connector Version | Nebula Algorithm Version | Nebula Version |
-|:-----------------------:|:------------------------------:|:------------------------:|:--------------:|
-|       2.0.0             |           2.0.0                |       2.0.0              |  2.0.0, 2.0.1  |
-|       2.0.1             |           2.0.1                |        --                |  2.0.0, 2.0.1  |
-|       2.1.0             |           2.1.0                |       2.1.0              |  2.0.0, 2.0.1  |
-|       2.5.0             |           2.5.0                |       2.5.0              |     2.5.0      |
-|     2.5-SNAPSHOT        |         2.5-SNAPSHOT           |       2.5-SNAPSHOT       |     nightly    |
+为了更好地进行产品的版本发布和管理，我们决定将 nebula-spark-utils 拆分成三个仓库，如果您要使用以上三个产品，请转移到产品对应的新仓库：
 
-If the Nebula version and the uitls' version are not in the same line, then they are incompatible.
-For Nebula Graph v2.x, see [nebula](https://github.com/vesoft-inc/nebula).
+* Nebula Exchange: https://github.com/vesoft-inc/nebula-exchange
+* Nebula Spark Connector: https://github.com/vesoft-inc/nebula-spark-connector
+* Nebula Algorithm: https://github.com/vesoft-inc/nebula-algorithm
+
+------
+
+> **Note**: 
+
+nebula-spark-utils repository contains three nebula products: Nebula Exchange, Nebula Spark Connector, Nebula Algorithm.
+
+In order to make product management more convenient, we decided to split the three products into three repositories. 
```

Comment:
In order for better version release and management, the team decided to split the current repo into three independent repos.